tests/virt_autotest/libvirt_isolated_virtual_network.pm

```diff
@@ -83,7 +83,7 @@ sub post_fail_hook {
     $self->SUPER::post_fail_hook;
 
     #Restart libvirtd service
-    virt_autotest::utils::restart_libvirtd();
+    virt_autotest::utils::restart_libvirtd() if is_monolithic_libvirtd;
```

Comment:
For this libvirt restart operation, I guess the reason is that it is in post_fail_hook, so possible that libvirtd run into some problematic state, to let the destroy_virt_network handled successfully, restart libvirtd may work, to achieve a good env cleanup.

Reply:
I'll create a new function named restart_modular_libvirt_daemons() function which will restart any daemons specified by parameters, fg. restart_modular_libvirt_daemons(virtqemud), restart all 7 daemons without any parameters passed.

Then the test owner will decide if it should be called and how it is called in their tests in the next release.

Reply:
Good to have default to all if nothing specified :)

Reply:
I created a ticket https://progress.opensuse.org/issues/129086 for further modular adaption in sle.